docs/guides/fine_tuning.md

```diff
@@ -110,8 +110,8 @@ validation data file, an optional set of hyperparameters to customize the fine-t
 and an optional suffix to append to the name of the fine-tune. For sequences longer than the native 
 `max_seq_length` of the model, the sequences will be truncated.
 
-If you specify a suffix, the fine-tune will be named `model:suffix:<timestamp>`. If you do not, 
-the fine-tune will be named `model:<timestamp>`. The timestamp will be the time the fine-tune was 
+If you specify a suffix, the fine-tune will be named `model.suffix.<timestamp>`. If you do not, 
```

Comment:
I asked @felixs8696 to change this to colons in the implementation btw.

Reply:
Colons breaks serving fine-tuned models, which is why we switched it back to periods.